core/jvmTest/src/kotlinx/serialization/features/JvmThirdPartyGenericsTest.kt

```diff
@@ -0,0 +1,19 @@
+package kotlinx.serialization.features
+
+import kotlinx.serialization.serializer
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+open class JvmThirdPartyGenericsTest : ThirdPartyGenericsTest() {
```

Comment:
I think there's no need in inheritance for nested classes to be accessible. Also, `open` modifier here for `JvmThirdPartyGenericsTest ` is definitely redundant

Reply:
> I think there's no need in inheritance for nested classes to be accessible.

This is true, but it also accesses instance vals `boxWithItemSerializer` and `serializersModule`. Thought inheritance was better than repetition in this case, but also happy to copy-paste these vals to this test if you prefer.

Remove `open`.

Reply:
OK, good point